### PR TITLE
feat: rewrite self-hosted install guide for Linux.

### LINKED ISSUE #2873: [Site Bug] Issue with the page: /self-hosted/latest/install/installation-linux/
_Use this template for reporting bugs in the docs._

# Describe the bug

-You have to add sudo in front of the commands. For the apt  commands and /usr/share/postgresql-common/pgdg/apt.postgresql.org.sh script.

-wget --quiet -O - https://packagecloud.io/timescale/timescaledb/gpgkey | sudo apt-key add -

This command fails for me on Ubuntu 20.04 and --quiet masks the error. The error is:

ERROR: cannot verify packagecloud.io's certificate, issued by ‘CN=Amazon RSA 2048 M01,O=Amazon,C=US’:
  Unable to locally verify the issuer's authority.
To connect to packagecloud.io insecurely, use `--no-check-certificate'.

I added --no-check-certificate to get past that, but that's far from ideal!

## What should the docs say?

# Page affected

[Provide the URL to the page you noticed the bug on]

# Subject matter expert (SME)

[If known, who is a good person to ask about this topic]

# Screenshots

[Attach images of screenshots showing the bug]

# Any further info

[Anything else you want to add, or further links]

## Contributing to documentation

We welcome documentation contributions!

*   For information about how to propose a change, see the [contributing guide](https://github.com/timescale/docs/blob/latest/CONTRIBUTING.md) in our GitHub repository.
*   For information on style and word usage, see the [style guide](https://docs.timescale.com/about/latest/contribute-to-docs)

### PR DESCRIPTION
**Pull request by billy-the-fish**
# Description

[Short summary of why you created this PR]

# Links

Fixes #[insert issue link, if any]

# Writing help

For information about style and word usage, see the [style guide](https://docs.timescale.com/about/latest/contribute-to-docs/)

# Review checklists

Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

*   [ ] Is the content technically accurate?
*   [ ] Is the content complete?
*   [ ] Is the content presented in a logical order?
*   [ ] Does the content use appropriate names for features and products?
*   [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

*   [ ] Is the content free from typos?
*   [ ] Does the content use plain English?
*   [ ] Does the content contain clear sections for concepts, tasks, and references?
*   [ ] Have any images been uploaded to the correct location, and are resolvable?
*   [ ] If the page index was updated, are redirects required
      and have they been implemented?
*   [ ] Have you checked the built version of this content?

### CODE REVIEW

**Review by jonatas — COMMENTED** [review]
any reason to not go with latest postgresql? because we're on  16 and some of the versions mentions 14?

**Comment by billy-the-fish** [comment]
Hi @jonatas, nice spot, thank you. Fixed.

File: _partials/_add-timescaledb-to-a-database.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+1. **Connect to your PostgreSQL instance**
+
+    ```bash
+    psql -U postgres -h localhost
+    ```
+
+1. **Create a database**
+
+    ```sql
+    CREATE database tsdb;
```
Comment (jonatas) [inline _partials/_add-timescaledb-to-a-database.md]: are we  going to use an official SQL style for the commands?

```suggestion
    CREATE DATABASE tsdb;
```

File: _partials/_install-self-hosted-debian-based.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+1.  **Install TimescaleDB**
+
+    ```bash
+    sudo apt install timescaledb-2-postgresql-16 postgresql-client
+    ```
+    
+    To install a specific TimescaleDB [release][releases-page], set the version. For example:
+    
+    `sudo apt-get install timescaledb-2-postgresql-12='2.6.0*' timescaledb-2-loader-postgresql-12='2.6.0*'`
```
Comment (jonatas) [inline _partials/_install-self-hosted-debian-based.md]: Can we suggest at least pg14? Timescale supports always the last 3 major versions.

File: _partials/_where-to-next.md [hunk 3]
```diff
@@ -1,9 +1,7 @@
-Now that you have your first Timescale database up and running, you can check
-out the [Use Timescale][tsdb-docs] section, and find out what
-you can do with it.
+Now you have TimescaleDB running, have a look at the:
 
-If you want to work through some tutorials to help you get up and running with
-Timescale and time-series data, check out the [tutorials][tutorials] section.
+* [Tutorials][tutorials]: walk through a variety of business scenarios using example datasets,
```
Comment (jonatas) [inline _partials/_where-to-next.md]:
```suggestion
* [Tutorials][tutorials]: walk through a variety of business scenarios using example datasets.
```

**Review by jonatas — APPROVED** [review]
Great work @billy-the-fish !

I haven't followed the steps.

**Pull request merged**